scheduler/drilling-wits-depth-median/src/api.py

```diff
@@ -0,0 +1,75 @@
+from typing import List
+
+import pydantic
+from corva import Api
+
+from src import models
+from src.configuration import SETTINGS
+
+
+def get_records(
+    api: Api,
+    asset_id: int,
+    log_identifier: str,
+    top_depth: float,
+    bottom_depth: float,
+) -> List[models.Record]:
+    raw_records = api.get_dataset(
+        provider='corva',
+        dataset=SETTINGS.IN_DATASET,
+        query={
+            'asset_id': asset_id,
+            'log_identifier': log_identifier,
+            'measured_depth': {'$gt': top_depth, '$lte': bottom_depth},
+        },
+        sort={'measured_depth': 1},
+        limit=SETTINGS.API_LIMIT,
+        fields='measured_depth,data',
+    )
+
+    return pydantic.parse_obj_as(List[models.Record], raw_records)
```

Comment:
Why isn't the API automatically returning as a List of Records?

Reply:
Good idea. https://corvaqa.atlassian.net/browse/DC-2005